ext/OceananigansNCDatasetsExt.jl

```diff
@@ -374,6 +374,22 @@ function gather_immersed_boundary(grid::GFBIBG, indices, dim_name_generator)
                 "immersed_boundary_mask_ccf" => Field(op_mask_ccf; indices))
 end
 
+const PCBIBG = ImmersedBoundaryGrid{<:Any, <:Any, <:Any, <:Any, <:Any, <:PartialCellBottom}
+
+# For Immersed Boundary Grids (IBG) with a Partial Cell Bottom (PCB)
+function gather_immersed_boundary(grid::PCBIBG, indices, dim_name_generator)
+    op_mask_ccc = KernelFunctionOperation{Center, Center, Center}(peripheral_node, grid, Center(), Center(), Center())
+    op_mask_fcc = KernelFunctionOperation{Face, Center, Center}(peripheral_node, grid, Face(), Center(), Center())
+    op_mask_cfc = KernelFunctionOperation{Center, Face, Center}(peripheral_node, grid, Center(), Face(), Center())
+    op_mask_ccf = KernelFunctionOperation{Center, Center, Face}(peripheral_node, grid, Center(), Center(), Face())
+
+    return Dict("bottom_height" => Field(grid.immersed_boundary.bottom_height; indices),
+                "immersed_boundary_mask_ccc" => Field(op_mask_ccc; indices),
+                "immersed_boundary_mask_fcc" => Field(op_mask_fcc; indices),
+                "immersed_boundary_mask_cfc" => Field(op_mask_cfc; indices),
+                "immersed_boundary_mask_ccf" => Field(op_mask_ccf; indices))
```

Comment:
did you mean to include this in the PR?

Reply:
I had the same question. Otherwise looks good!

Reply:
Ugh, second time I add extra code by mistake. Thanks for catching that!